lib/tls.js

```diff
@@ -174,13 +171,11 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 
     if (!valid) {
       if (cert.subjectaltname) {
-        reason = util.format('Host: %s is not in the cert\'s altnames: %s',
-                             host,
-                             cert.subjectaltname);
+        reason =
+            `Host: ${host} is not in the cert's altnames:` +
+            `${cert.subjectaltname}`;
```

Comment:
You're missing a space between the `:` and the `cert.subjectaltname` in the replacement text